tedana/tests/test_integration.py

```diff
@@ -103,6 +102,38 @@ def test_integration_five_echo(skip_integration):
     check_integration_outputs(fn, out_dir)
 
 
+def test_integration_four_echo(skip_integration):
+    """ Integration test of the full tedana workflow using four-echo test data
+    """
+
+    if skip_integration:
+        pytest.skip('Skipping four-echo integration test')
+    out_dir = '/tmp/data/four-echo/TED.four-echo'
+    if os.path.exists(out_dir):
+        shutil.rmtree(out_dir)
+
+    # download data and run the test
+    download_test_data('https://osf.io/gnj73/download',
+                       os.path.dirname(out_dir))
+    prepend = '/tmp/data/four-echo/'
+    prepend += 'sub-PILOT_ses-01_task-localizerDetection_run-01_echo-'
+    suffix = '_space-sbref_desc-preproc_bold+orig.HEAD'
+    datalist = [prepend + str(i + 1) + suffix for i in range(4)]
+    tedana_workflow(data=datalist,
+                    tes=[11.8, 28.04, 44.28, 60.52],
+                    out_dir=out_dir,
+                    tedpca='mdl')
```

Comment:
5-echo was switched to 'mdl' in this branch so might as well do 'aic' or something else.